src/main/java/eu/enmeshed/exception/status/EnmeshedHttpStatus.java

```diff
@@ -1,13 +1,6 @@
 package eu.enmeshed.exception.status;
 
-import lombok.AllArgsConstructor;
 import lombok.Builder;
-import lombok.Getter;
 
-@Getter
-@AllArgsConstructor
 @Builder
-public class EnmeshedHttpStatus {
-  private final int errorCode;
-  private final String message;
-}
+public record EnmeshedHttpStatus(int errorCode, String message) {}
```

Comment:
Since I am not a Java expert: Why does this change to `EnmeshedHttpStatus` does not lead to the need of a change within the `EnmeshedHttpStatuses.java` file?

Reply:
it's just a shorthand